### DISCUSSION TITLE
Approval Request for Transition Plan

### DISCUSSION THREAD

**Discussion by vvalderrv** [NativeScript/NativeScript#10559] [General]
The Linux Foundation is seeking approval of the transition plan outlined in the document linked below:

[NativeScript Transition Plan](https://docs.google.com/spreadsheets/d/1wS5jzC0fgVjetQI0sql2CyLge8NRk18q3gnsIHf10lo/edit?usp=sharing)

[NativeScript Future State Approval](https://github.com/orgs/NativeScript/discussions/10502) for reference

Please review and sign off on the plan by commenting with your approval.

If you have any further questions, please review the [Transition FAQ](https://docs.google.com/document/d/1gESWS5brgxXm9EQrxctmievRedyYp9Q0dtHxLEcL1nk/edit?usp=sharing) and feel free to reach out directly.

Upon approval, The Linux Foundation will work with the NativeScript community on timelines for transition.

Thank you for your attention to this matter.
Vanessa Valderrama
LF Release Engineering
[vvalderrama@linuxfoundation.org](mailto:vvalderrama@linuxfoundation.org)

**Comment by NathanWalker** [comment]
@vvalderrv Approved. We'll let @shirakaba provide approval as well.

**Reply by shirakaba** [reply]
@vvalderrv I've requested access to the Transition Plan document.

**Reply by shirakaba** [reply]
I now have access to the document 👍

@vvalderrv approved by the TSC Chair (me)!

**Reply by shirakaba** [reply]
Hi @vvalderrv how is this progressing?
